examples/portfolio/src/content/config.ts

```diff
@@ -1,6 +1,7 @@
 import { defineCollection, z } from 'astro:content';
 
 export const collections = {
+	type: 'content',
```

Comment:
It should be on line 5 instead

Reply:
Thanks for the review! This should be fixed now.